### ISSUE TITLE
0.7.0 Installation windi.css resolution issue

### ISSUE THREAD

**Issue by timothyallan** [thisbeyond/solid-select#8]
Heya, just did a fresh install, and simply doing `npm run dev` on my basic Tailwind Solid Template app with an `import { Select } from '@thisbeyond/solid-select';` gets me:

``` 
[vite] Internal server error: Failed to resolve import "virtual:windi.css" from "node_modules/@thisbeyond/solid-select/dist/source/index.jsx". Does the file exist?
```

Possibly relevant deps are: 
```
    "vite": "^2.8.6",
    "vite-plugin-solid": "^2.2.6",
    "solid-js": "^1.3.13",
```

**Comment by martinpengellyphillips** [comment]
Ah, yeah - this is because we export the 'source' for use in other Solid project builds, but you won't necessarily have WindiCSS to generate the stylesheet. 

I think I can handle this by just removing the offending line as part of the build process - I'll give that a go. 

Longer term there is likely a better way to manage the styling in general.

**Comment by timothyallan** [comment]
Aha, it's not a bundled output, got it. Okay, will sort out how/if I can install WindiCSS with my current Tailwind setup. Thanks!

**Comment by martinpengellyphillips** [comment]
Just for clarity - there is a bundled output as well (`dist/esm`), but because there is also the 'solid' export of the JSX source (as seems to be convention for Solid libraries) it defaults to that when used in another Solid project.

I've just released [0.7.1](https://github.com/thisbeyond/solid-select/releases/tag/0.7.1) which should fix the issue for you with no need to add WindiCSS to your setup. Please test and let me know!

**Comment by timothyallan** [comment]
0.7.1 seems to have fixed it up, nice one!
